src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
```

Comment:
The @Data annotation includes @Getter, @Setter, @ToString, and @EqualsAndHashCode. Using @Data may introduce unnecessary methods and increase the class complexity.